app/Console/Kernel.php

```diff
@@ -26,7 +26,6 @@ class Kernel extends ConsoleKernel
         \Ushahidi\Console\Command\ApikeySet::class,
         \Ushahidi\Console\Command\ConfigSet::class,
         \Ushahidi\Console\Command\ConfigGet::class,
-        \Ushahidi\Console\Command\Import::class,
         \Ushahidi\Console\Command\UserCreate::class,
```

Comment:
I don't remember why we needed to delete this :/ https://github.com/ushahidi/platform/pull/3430/files I wish I had notes on the decision leading to this change . Note to review after the rest of the code

Reply:
OK so it looks like this change was a pre requirement for this , which is moving the imports to queues and making them more useful in the process https://github.com/ushahidi/platform/pull/3407/files

But this means we probably want to roll it back until we have the new system with queuws in place and the necessary UI changes so that the user understands what's happening. @tuxpiper thoughts?

Reply:
We do have platform api queue worker instances for ushahidi.io (in production and steve-buscemi) running `artisan queue:work` (for the exports). I think the import stuff should fit right in :+1:

Reply:
ah! alright. Does that mean we could merge the import queues code then?

Reply:
I think we should definitely try! Is the plan to merge to master and then QA , or do some QA on develop first? I would suggest the latter if possible.

Reply:
Let's do develop first if we plan to merge the import queues work https://github.com/ushahidi/platform/pull/3407/files

Reply:
actually let's do it in develop either way :D at least a smoke test and then a regression in master?

Reply:
since develop is continuously deployed to steve-buscemi , we should be good to do that smoke test anytime